modules/_node-scripts/esbuild/getImportBridgesPlugin.mjs

```diff
@@ -0,0 +1,79 @@
+import {IMPORT_BRIDGE_FILTER, decodeBridgePath} from './getImportBridgePath.mjs';
+import getExportedSymbols from '../util/getExportedSymbols.mjs';
+import getFlatName from '../util/getFlatName.mjs';
+import getPathPrefix from './getPathPrefix.mjs';
+
+export default function getImportBridgesPlugin(globalImports, globalSymbols) {
+	const importBridgesCache = {};
+
+	return {
+		name: 'import-bridges-plugin',
+
+		setup(build) {
+			build.onLoad(
+				{ 
+					filter: IMPORT_BRIDGE_FILTER 
+				}, 
+				async (args) => {
+					const {path: loadPath} = args;
+
+					if (importBridgesCache[loadPath]) {
+						return importBridgesCache[loadPath];
+					}
+
+					const {moduleName, type} = decodeBridgePath(loadPath);
+
+					const {external, webContextPath} = globalImports[moduleName];
+
+					const contents = getImportBridgeCode(
+						globalImports, globalSymbols, type, moduleName, external, 
+						webContextPath
+					);	
+
+					importBridgesCache[loadPath] = {
+						contents,
+						loader: 'js',
+					};
+
+					return importBridgesCache[loadPath];
+				}
+			);
+		},
+	};
+}
+
+function getImportBridgeCode(
+	globalImports, globalSymbols, type, moduleName, external, webContextPath
+) {
+	let hasDefault;
+
+	if (globalImports[moduleName]?.external === false) {
+		hasDefault = false;
+	}
+	else {
+		const symbols = getExportedSymbols(globalSymbols, moduleName);
+
+		hasDefault = !!symbols['default'];
+	}
+
+	if (moduleName === 'react' || moduleName === 'react-dom' || moduleName === 'object-hash' || moduleName === 'prop-types') {
```

Comment:
Is this going to end up growing? What if we add a new dependency that has a default export, will it need to be added here?

Reply:
:scream:  That's a leftover from previous tests. That is actually inferred using the exported package symbols (either by autodetecting them or using the overriden package symbols in configuration).

Reply:
done